module/Company/src/Company/Service/Company.php

```diff
@@ -245,161 +245,153 @@ public function getLabelList($visible)
     }
 
     /**
-     * Inserts a category, and binds it to the given package
+     * Creates a new JobCategory.
      *
-     * @param mixed $packageID
+     * @param  array $data Category data from the EditCategory form.
+     * @return bool|int Returns false on failure, and the languageNeutralId on success
      */
-    public function insertCategory($lang, $id, $category = null)
+    public function createCategory($data)
     {
-        if (!$this->isAllowed('insert')) {
```

Comment:
Why did we remove this assertion?

Reply:
I have missed that when re-ordering the code. I will add it back later today.